radix-engine/src/engine/interpreters/wasm_runtime.rs

```diff
@@ -186,6 +186,10 @@ where
         return Ok(self.actor.clone());
     }
 
+    fn handle_get_transaction_hash(&mut self) -> Result<Hash, RuntimeError> {
```

Comment:
Can we move this into another RENode rather than a separate system call? It would be a RENode which contains other info like time/validator proposing/etc.